web-common/src/features/users/roles.ts

```diff
@@ -0,0 +1,6 @@
+export enum UserRoles {
```

Comment:
I can see that most of the changes are using an enum for roles. The reason it wasn't consolidated was that the project doesn't have a guest role. We can perhaps split into `OrgUserRoles` and `ProjectUserRoles` for clarity. A single generic enum will introduce an extra mental overload in understanding if they were for org or project.

Reply:
There is quite a bit of mixing of roles. Lemme know if the changes are correct.

Reply:
Took a look at the refactors with OrgUsersRoles and ProjectUserRoles. Thanks for the change!